src/pages/studyView/table/StructuralVariantMultiSelectionTable.tsx

```diff
@@ -0,0 +1,782 @@
+    toggleSelectionOperator() {
+        const selectionType = this._selectionType || this.selectionType;
+        if (selectionType === SelectionOperatorEnum.INTERSECTION) {
+            this._selectionType = SelectionOperatorEnum.UNION;
+        } else {
+            this._selectionType = SelectionOperatorEnum.INTERSECTION;
+        }
+        localStorage.setItem(this.props.tableType, this.selectionType);
+    }
+
+    @autobind
+    isSelectedRow(data: StructVarMultiSelectionTableRow) {
+        return this.isChecked(data.uniqueKey);
+    }
+
+    @computed get filterKeyToIndexSet() {
+        return _.reduce(
+            this.props.filters,
+            (acc, next, index) => {
+                next.forEach(key => {
+                    acc[key] = index;
+                });
+                return acc;
+            },
+            {} as { [id: string]: number }
+        );
+    }
+
+    @autobind
+    selectedRowClassName(data: StructVarMultiSelectionTableRow) {
+        const index = this.filterKeyToIndexSet[data.uniqueKey];
+        if (index === undefined) {
+            return this.props.filters.length % 2 === 0
+                ? styles.highlightedEvenRow
+                : styles.highlightedOddRow;
+        }
+        return index % 2 === 0
+            ? styles.highlightedEvenRow
+            : styles.highlightedOddRow;
+    }
+
+    @action.bound
+    afterSorting(
+        sortBy: StructVarMultiSelectionTableColumnKey,
+        sortDirection: SortDirection
+    ) {
+        this.sortBy = sortBy;
+        this.sortDirection = sortDirection;
+    }
+
+    public render() {
+        const tableId = `${this.props.tableType}-table`;
+        return (
+            <div data-test={tableId} key={tableId}>
+                {this.props.promise.isComplete && (
+                    <MultiSelectionTableComponent
+                        width={this.props.width}
+                        height={this.props.height}
+                        data={this.selectableTableData}
+                        columns={this.tableColumns}
+                        isSelectedRow={this.isSelectedRow}
+                        afterSelectingRows={this.afterSelectingRows}
+                        defaultSelectionOperator={this.selectionType}
+                        toggleSelectionOperator={this.toggleSelectionOperator}
+                        extraButtons={this.props.extraButtons}
+                        sortBy={this.sortBy}
+                        sortDirection={this.sortDirection}
+                        afterSorting={this.afterSorting}
+                        fixedTopRowsData={this.preSelectedRows}
+                        highlightedRowClassName={this.selectedRowClassName}
+                        showSetOperationsButton={true}
+                        numberOfSelectedRows={
+                            this.props.selectedRowsKeys.length
+                        }
+                    />
+                )}
+                {this.props.genePanelCache ? (
```

Comment:
maybe add modalOpen check to the initial condition so that we don't bother rendering the modal in hidden state.   then make the "show" attribute of Modal always = true.

Reply:
This code is again directly copied from MultiSelectionTable.tsx.